akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ConsumerControllerImpl.scala

```diff
@@ -228,6 +228,8 @@ private class ConsumerControllerImpl[A](
 
   private val flightRecorder = ActorFlightRecorder(context.system).delivery
 
+  private val traceEnabled = context.log.isTraceEnabled
```

Comment:
as this is typed and slf4j directly this log level can be changed dynamically at runtime which wouldn't be reflected here

Reply:
correct, but I thought that was an ok limitation (enabling trace in production is probably not a good idea anyway)

Reply:
Fair enough!